evap/student/templates/student_index.html

```diff
@@ -81,34 +81,44 @@
                                             {% endif %}
                                         {% endif %}
                                     </td>
-                                    <td class="text-right">
-                                        {% if course.grades_activated and course.grade_documents.count == 1 and can_download_grades %}
-                                            <a class="btn btn-sm btn-light" href="{% url 'grades:download_grades' course.grade_documents.all.first.id %}" role="button">{{ course.grade_documents.all.first.description }}</a>
-                                        {% elif course.grade_documents.count > 1 and can_download_grades %}
-                                            <div class="btn-group" role="group">
-                                                <a id="btnDownload{{ course.id }}" role="button" class="btn btn-sm btn-light dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">{% trans 'Grades' %}</a>
-                                                <div class="dropdown-menu" aria-labelledby="btnDownload{{ course.id }}">
+                                    {% if can_download_grades %}
+                                        <td class="text-center">
+                                            {% if course.grades_activated and course.grade_documents.count == 1 %}
+                                                <a class="btn btn-sm btn-sm-narrow btn-light" href="{% url 'grades:download_grades' course.grade_documents.all.first.id %}" role="button"
+                                                    data-toggle="tooltip" data-placement="top" title="{{ course.grade_documents.all.first.description }}">
+                                                    <span class="far fa-file"></span>
+                                                </a>
+                                            {% elif course.grades_activated and course.grade_documents.count > 1 %}
+                                                <div class="btn-group" role="group">
                                                     {% for grade_document in course.grade_documents.all %}
```

Comment:
Afaik, course.grade_documents.all is a function and should be called here, shouldn't it?
Same for the `count`s, `all`s and `first`s above. Also, some of the `.all().first()` could be shortened to just `first()`.

Reply:
function calls are written without parentheses in django templates. you're right about the `.all.first`.

Reply:
Ah, well, forgot about that. I'm fine with the code, then :)